visualizations/frontend/other/TTable.vue

```diff
@@ -380,6 +376,38 @@ export default {
     uniqueOriginalRows() {
       return _.uniq(this.internalRows.map((r) => r.originalRow));
     },
+    layerRows() {
+      if (typeof this.rows === "function")
+        return this.rows(this.$attrs?.["t-layer"]);
+      if (this.rows) return this.rows;
+      return [];
+    },
+  },
+  watch: {
+    loading() {
```

Comment:
This might be a super silly question. 

What 'loading' are we watching? I don't see a data property called 'loading'.

Reply:
That is not a silly question at all, it is provided by the back end with a [mixin](https://v2.vuejs.org/v2/guide/mixins.html?redirect=true). Looks like it is a computed property in layer_mixin.js but I haven't tested that.

Reply:
oh mixins. The trick me every time.

Reply:
same... I love the idea but magic functionality you can't see? Hard to reason about 🙄